pkg/admission/initializers/interfaces.go

```diff
@@ -29,6 +30,12 @@ type WantsKcpInformers interface {
 	SetKcpInformers(informers kcpinformers.SharedInformerFactory)
 }
 
+// WantsKubeInformers interface should be implemented by admission plugins that want to have a kube informer factory
+// injected.
+type WantsKubeInformers interface {
+	SetKubeInformers(informers informers.SharedInformerFactory)
+}
```

Comment:
this definitely exists upstream

Reply:
WantsExternalKubeInformerFactory

Reply:
Fixed in squash commit

Reply:
Actually, this requires instantiating one of https://github.com/kcp-dev/kubernetes/blob/5bb0eeecf2cf172e4fc8b414c1953e64634a2a6a/staging/src/k8s.io/apiserver/pkg/admission/initializer/initializer.go#L27-L70, is that ok?

Reply:
I got rid of our custom interface, but retained the custom impl. I don't want to init the upstream combo initializer.